Jenkinsfile.ci

```diff
@@ -0,0 +1,100 @@
+#!groovy
+
+try {
```

Comment:
This try/catch and testing function look redundant now. We can just content of testing function at the top level,